src/command_modules/azure-cli-servicebus/azure/cli/command_modules/servicebus/custom.py

```diff
@@ -41,6 +49,12 @@ def cli_namespace_update(instance, tags=None, sku=None, capacity=None):
     if capacity is not None:
         instance.sku.capacity = capacity
 
+    if default_action:
+        resourcegroup = instance.id.split('/')[4]
```

Comment:
better would be to use the `parse_resource_id` method from msrestazure.tools, which returns a dictionary.

Reply:
used 'parse_resource_id' method,  Resolved